src/tree/SubscriptionTreeItem.ts

```diff
@@ -207,3 +178,69 @@ async function getDefaultFuncVersion(context: ICreateFunctionAppContext): Promis
 
     return version;
 }
+
+async function createFunctionAppChild(wizardContext: IFunctionAppWizardContext, promptSteps: AzureWizardPromptStep<IAppServiceWizardContext>[], executeSteps: AzureWizardExecuteStep<IAppServiceWizardContext>[]) {
+    promptSteps.push(new FunctionAppStackStep());
+
+    const storageAccountCreateOptions: INewStorageAccountDefaults = {
+        kind: StorageAccountKind.Storage,
+        performance: StorageAccountPerformance.Standard,
+        replication: StorageAccountReplication.LRS
+    };
+
+    if (wizardContext.version === FuncVersion.v1) { // v1 doesn't support linux
+        wizardContext.newSiteOS = WebsiteOS.windows;
+    }
+
+    if (!wizardContext.advancedCreation) {
+        LocationListStep.addStep(wizardContext, promptSteps);
+        wizardContext.useConsumptionPlan = true;
+        wizardContext.stackFilter = getRootFunctionsWorkerRuntime(wizardContext.language);
+        executeSteps.push(new ResourceGroupCreateStep());
+        executeSteps.push(new AppServicePlanCreateStep());
+        executeSteps.push(new StorageAccountCreateStep(storageAccountCreateOptions));
+        executeSteps.push(new LogAnalyticsCreateStep());
+        executeSteps.push(new AppInsightsCreateStep());
+    } else {
+        promptSteps.push(new ResourceGroupListStep());
+        CustomLocationListStep.addStep(wizardContext, promptSteps);
+        promptSteps.push(new FunctionAppHostingPlanStep());
+        promptSteps.push(new StorageAccountListStep(
+            storageAccountCreateOptions,
+            {
+                // The account type must support blobs, queues, and tables.
+                // See: https://aka.ms/Cfqnrc
+                kind: [
+                    // Blob-only accounts don't support queues and tables
+                    StorageAccountKind.BlobStorage
+                ],
+                performance: [
+                    // Premium performance accounts don't support queues and tables
+                    StorageAccountPerformance.Premium
+                ],
+                learnMoreLink: 'https://aka.ms/Cfqnrc'
+            }
+        ));
+        promptSteps.push(new AppInsightsListStep());
+    }
+
+    executeSteps.push(new FunctionAppCreateStep());
+}
+
+async function createContainerizedFunctionAppChild(wizardContext: IFunctionAppWizardContext, promptSteps: AzureWizardPromptStep<IAppServiceWizardContext>[], executeSteps: AzureWizardExecuteStep<IAppServiceWizardContext>[]) {
+    const storageAccountCreateOptions: INewStorageAccountDefaults = {
+        kind: StorageAccountKind.Storage,
+        performance: StorageAccountPerformance.Standard,
+        replication: StorageAccountReplication.LRS
+    };
+
+    LocationListStep.addStep(wizardContext, promptSteps);
```

Comment:
The locations listed in the prompt also appear on the locations list in the portal when you choose to deploy from image when creating a function app. I would think that would mean they should work?

Reply:
Well, I tried West India and it didn't successfully create. We may not be able to filter it if the service doesn't recognize it will fail until runtime though.